install.sh

```diff
@@ -49,16 +52,68 @@ else
 	fi
 fi
 
+
+allBuildArguments=("$@")
+for ((index=0; index < ${#allBuildArguments[@]}; index++)); do
+	if [ ${allBuildArguments[index]} = '-r' ]; then
+		reposArg=${allBuildArguments[index+1]}
+		if [ "$reposArg" = "" ]; then
+			echo "Missing required argument for -r"
+			exit 1 # exit script with error
+		fi
+		# parse the r option values
+		IFS=',' read -r -a repos <<< "$reposArg"
+	fi
+done
+
 if [ -d ../operator ]; then 
 
 	# pluck image tag off operator image 
 	tag=$(cat ../operator/kappnav.yaml | grep operator: | awk '{ split($0,p,":"); print p[3] }')
-
+	
 	echo Install kappnav to kubeenv $kubeenv
 	kubectl create namespace kappnav 
 
-	cat ../operator/kappnav.yaml | sed "s|kubeEnv: okd|kubeEnv: $kubeenv|" | sed "s|repository: kappnav/|repository: $org/kappnav-|" | sed "s|tag: $tag|tag: latest|" | sed "s|image: kappnav/operator:$tag|image: $org/kappnav-operator:latest|" | kubectl create -f - -n kappnav 
+	if [[ x$reposArg == 'x' || x$reposArg == "" ]]; then
```

Comment:
The expression on the right side of the `||` is always false.

Reply:
can you pls tell me the line number of the expression you are referring to?

Reply:
77